content/content-strategy/index.yml

```diff
@@ -9,23 +9,12 @@ main:
   - why.md
   - how-head.md
   - how.md
-  - begin-head.md
   - begin.md
   - begin-cards.md
-  - research-head.md
-  - research.md
   - research-cards.md
-  - develop-head.md
-  - develop.md
   - develop-cards.md
-  - clean-head.md
-  - clean.md
   - clean-cards.md
-  - set-foundations-head.md
-  - set-foundations.md
   - foundations-cards.md
-  - create-ucc-head.md
-  - create-ucc.md
```

Comment:
👍 looks good to me! Thanks for replacing.

Reply:
It's all merged together into the cards_with_intro content. Makes it easy for the content editor